notify/telegram/telegram.go

```diff
@@ -44,6 +45,19 @@ func New(conf *config.TelegramConfig, t *template.Template, l log.Logger, httpOp
 		return nil, err
 	}
 
+	if conf.APIUrl == nil {
+		apiURL, err := url.Parse("https://api.telegram.org")
```

Comment:
nit: Can this be a `const`?

Reply:
This should be done during unmarhalling of the configuration, not when calling New.

Reply:
Good shout Julien.

On a second look at https://github.com/prometheus/alertmanager/blob/71d61c9c9149420209c973014e0d6c981b183611/config/notifiers.go#L140-L147

It seems like both of these belong within `DefaultTelegramConfig`

Reply:
nit really because the API should come from DefaultGlobalConfig